interface.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export interface ITodoList {
+  contents: string;
+  id: number;
+  isDone: boolean;
+}
+export interface IYetListProps {
+  list: [];
```

Comment:
어떤 array인지 좀더 정확하게...!
```suggestion
  list: ITodoList;
```